test/complex.jl

```diff
@@ -15,6 +15,21 @@ end
 @test sprint(show, complex(1, 0), context=:compact => true) == "1+0im"
 @test sprint(show, complex(true, true)) == "Complex(true,true)"
 
+@testset "unary operator on complex boolean" begin
+    @test isequal(+(complex(true)), complex(1, 0))
+    @test isequal(+(complex(false)), complex(0, 0))
+    @test isequal(-(complex(true)), complex(-1, 0))
+    @test isequal(-(complex(false)), complex(0, 0))
```

Comment:
I think the ones above here using single-argument `complex` are now redundant since we're not testing the behavior of the `complex` function and `complex(true)` and `complex(true, false)` produce the same value. Also, it would be more correct for these tests to use the `Complex(true, false)` constructor, not the `complex` conversion.

Reply:
What do you say, I scrape them or should I test the `Complex(true)`?
As for the second point, are you suggesting `complex(true, false)` -> `Complex(true, false)` ?

Reply:
Just test `+` and `-` applied to the four `Complex(true, false)` values.

Reply:
Done!